test/vars/UtilSpec.groovy

```diff
@@ -435,4 +435,26 @@ class UtilSpec extends JenkinsPipelineSpecification {
         then:
         result == "/workspacefolder/kiegroup_projectA"
     }
+    def "[util.groovy] getNextSnapshot"() {
```

Comment:
We should add 4 more test cases

- major
- minor
- micro
- non of them (error expected)
- with suffix

Reply:
For micro it's already there, for major and minor I think so I have to add which I think to do in the already existing text will be good enough ,and for error things are already added in it. Once I push let me know if something is missed.

Reply:
with error I mean once you add the `assert` line